app/components/listing/LocationTimesAccordion.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { LocationDetails } from "models";
+import classNames from "classnames";
+import styles from "./LocationTimesAccordion.module.scss";
+
+/*
+    Accordion built per a11y guidelines:
+    https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+*/
+
+const LocationTimesAccordion = ({
+  locationRenderer,
```

Comment:
How many different renderers do we plan to use in practice? I've only found `<TableOfOpeningTimes />` in the code? What's the chance we can refactor this out to avoid the prop drilling?

Reply:
yes, good catch